repositories/device.go

```diff
@@ -75,11 +80,32 @@ func (m *DeviceModel) fromModel() twomes.Device {
 }
 
 func (r *DeviceRepository) Find(device twomes.Device) (twomes.Device, error) {
+	fmt.Println("Find device", device.ID)
 	deviceModel := MakeDeviceModel(device)
-	err := r.db.Preload("DeviceType").Preload("Uploads").Where(&deviceModel).First(&deviceModel).Error
+	err := r.db.Preload("DeviceType").Preload("Uploads").Preload("Uploads.Measurements").Where(&deviceModel).First(&deviceModel).Error
 	return deviceModel.fromModel(), err
 }
 
+func (r *DeviceRepository) GetProperties(device twomes.Device) ([]twomes.Property, error) {
+	var properties []twomes.Property = make([]twomes.Property, 0)
```

Comment:
Not a big deal, just a tip:
make(....., 0) does the same thing as just creating the var without make. This is only useful if you want to create an initial size (not 0).

Reply:
Thanks! I thought the variable would be `nil` if I did not use `make(...)`.